docs/en/UI/AspNetCore/Tag-Helpers/Badges.md

```diff
@@ -0,0 +1,38 @@
+# Badges
+
+## Introduction
+
+`abp-badge` and  `abp-badge-pill` are abp tags for badges.
+
+Basic usage:
+
+````csharp
+<span abp-badge="Primary">Primary</span>
```

Comment:
It can be used on `<a>` tag too.

Reply:
added two more basic samples including a tag